File: apzkr-pzpi-21-10-mykolchuk-mykhailo/Task4-MobileClient/app/src/main/java/com/example/apz_mobile/models/Sensor.kt
package com.example.apz_mobile.models

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Sensor (
    val id: Int,
    val name: String,
    val car: CarForSensor?
)  : Parcelable

@Parcelize
data class CarForSensor(
    val id: Int,
    val name: String,
    val type: String,
    val added: String,
    val drivers: List<Driver> = listOf(),
)  : Parcelable

File: apzkr-pzpi-21-10-mykolchuk-mykhailo/Task1-Server/Domain/Models/Sensor.cs
namespace APZ_backend.Models
{
    public class Sensor
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int OwnerId { get; set; }
        public User Owner { get; set; }
        public int? CarId { get; set; }
        public Car? Car { get; set; }
    }
}
